reference/features/telemetry.md

```diff
@@ -0,0 +1,50 @@
+# Telemetry
+
+MeiliSearch collects anonymized data from users in order to improve our product. This can be [deactivated at any time](#how-to-disable-data-collection).
+
+Note that while MeiliSearch is running in [`development` mode](/reference/features/configuration.md#environment), no data is collected.
+
+## What kind of data do you collect?
+
+We use two data-collection tools: [Amplitude](https://amplitude.com/amplitude-analytics) and [Sentry](https://sentry.io/welcome/). Both can be disabled by [following the below procedure](#how-to-disable-data-collection).
+
+**Amplitude** collects information about how and where MeiliSearch is being used, including the amount of time a MeiliSearch instance has been running, the version it's using, the device that's running it, and the number of documents per index. It also tracks the number of unique users, which is our equivalent of counting downloads or installs.
```

Comment:
Would a bulleted list be better? We could just copy and paste the list from @gmourier 's specification:
```
- Cohort: Servers that are running for more than seven days with the old version of MS (< v0.10.1)
- Cohort: Servers that are running for more than seven days with the new version of MS (> v0.10.1)
- Cohort: Servers that are running for more than 30 days with the old version of MS (< v0.10.1)
- Cohort: Servers that are running for more than 30 days with the new version of MS (> v0.10.1)
- Graph: The total of every telemetry messages received
- Graph: The total of every unique id seen
- Graph: The number of servers that are running at least since 7 days
- Graph: The number of servers that are running at least since 30 days
- User Composition: Device composition
- User Composition: Version composition
- Indexes size distribution for more than seven days (<200k not taken into account)
- Indexes size distribution for the others
```

Reply:
If we choose to go towards a bullet list (which is a good idea I think) we should be able to simplify it. The specification describes internal details at the Amplitude level on how we organize the metrics. Not sure if the details of Cohort, Graph etc.. are useful for the reader. In fact, I think the four lines about cohorts can be removed because the lines 
```
- Graph: The number of servers that are running at least since 7 days 
- Graph: The number of servers that are running at least since 30 days
```
are identical in terms of information for the end user.

If I'm not mistaken (@qdequele, please correct me if I'm wrong), a cohort is in fact a segment made up of "users" (MeiliSearch instance in our case) who share the same traits (e.g., active for more than 30 days or active for more than 7 days).  We can feed different tables or graphs to compare several cohorts.